lib/Github/Exception/ApiLimitExceedException.php

```diff
@@ -9,8 +9,24 @@
  */
 class ApiLimitExceedException extends RuntimeException
 {
-    public function __construct($limit = 5000, $code = 0, $previous = null)
+    private $limit;
+    private $reset;
+
+    public function __construct($limit = 5000, $reset = 1800, $code = 0, $previous = null)
```

Comment:
If this going to be only part of a new major release, this change is fine of course.